test/unit/ios-log-specs.js

```diff
@@ -55,4 +56,33 @@ describe('system logs', () => {
 
     await log.stopCapture();
   });
+
+  it('should rotate log buffer', async function () {
+    let log = new IOSLog({sim, showLogs: false});
+    log.logIdxSinceLastRequest.should.be.below(0);
+    await log.startCapture();
+    let recentLogs = log.getLogs();
+    recentLogs.should.have.lengthOf(0);
+    log.logIdxSinceLastRequest.should.be.below(0);
+
+    for (let i = 0; i <= MAX_LOG_ENTRIES_COUNT * 2; ++i) {
+      await fs.writeFile(tmpSystemLog, `${i}\n`, {flag: 'a'});
+      if (i === 0 || i % (MAX_LOG_ENTRIES_COUNT / 2) !== 0) {
+        continue;
+      }
+      // on some slow system (e.g., Travis) need a moment
+      await B.delay(3000);
+      recentLogs = log.getLogs();
+      recentLogs.should.have.length.within(1, MAX_LOG_ENTRIES_COUNT / 2);
```

Comment:
i don't understand why we want to assert on a range here? don't we know how many lines there should be?